Scripts/Modules/SnappingModule/SnappingModuleSettingsUI.cs

```diff
@@ -0,0 +1,88 @@
+﻿#if UNITY_EDITOR
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace UnityEditor.Experimental.EditorVR
+{
+	public class SnappingModuleSettingsUI : MonoBehaviour
+	{
+		[SerializeField]
+		Toggle m_SnappingEnabled;
+
+		[SerializeField]
+		Toggle m_GroundSnapping;
+
+		[SerializeField]
+		Toggle m_SurfaceSnapping;
+
+		[SerializeField]
+		Toggle m_PivotSnapping;
+
+		[SerializeField]
+		Toggle m_SnapRotation;
+
+		[SerializeField]
+		Toggle m_LocalOnly;
+
+		[SerializeField]
+		Toggle m_ManipulatorSnapping;
+
+		[SerializeField]
+		Toggle m_DirectSnapping;
+
+		public Toggle snappingEnabled
+		{
+			get { return m_SnappingEnabled; }
+		}
+
+		public Toggle groundSnapping
```

Comment:
This is minor, but you could be consistent and change all of these (e.g. groundSnapping) to xEnabled (e.g. groundSnappingEnabled).